src/pkg/packager/inspect.go

```diff
@@ -6,17 +6,68 @@ package packager
 
 import (
 	"fmt"
+	"path/filepath"
+	"strings"
 
 	"github.com/defenseunicorns/zarf/src/internal/packager/sbom"
+	"github.com/defenseunicorns/zarf/src/pkg/message"
 	"github.com/defenseunicorns/zarf/src/pkg/utils"
+	"github.com/mholt/archiver/v3"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/pterm/pterm"
+	"oras.land/oras-go/v2/registry"
 )
 
 // Inspect list the contents of a package.
 func (p *Packager) Inspect(includeSBOM bool, outputSBOM string) error {
+	if utils.IsOCIURL(p.cfg.DeployOpts.PackagePath) {
+		spinner := message.NewProgressSpinner("Loading Zarf Package %s", p.cfg.DeployOpts.PackagePath)
+		ref, err := registry.ParseReference(strings.TrimPrefix(p.cfg.DeployOpts.PackagePath, "oci://"))
+		if err != nil {
+			return err
+		}
+
+		dst, err := utils.NewOrasRemote(ref)
+		if err != nil {
+			return err
+		}
+
+		// get the manifest
+		spinner.Updatef("Fetching the manifest for %s", p.cfg.DeployOpts.PackagePath)
+		layers, err := getLayers(dst)
+		if err != nil {
+			return err
+		}
+		spinner.Updatef("Loading Zarf Package %s", p.cfg.DeployOpts.PackagePath)
+		zarfYamlDesc := utils.Find(layers, func(d ocispec.Descriptor) bool {
+			return d.Annotations["org.opencontainers.image.title"] == "zarf.yaml"
+		})
+		err = pullLayer(dst, zarfYamlDesc, p.tmp.ZarfYaml)
+		if err != nil {
+			return err
+		}
 
-	if err := p.loadZarfPkg(); err != nil {
-		return fmt.Errorf("unable to load the package: %w", err)
+		if includeSBOM {
```

Comment:
```suggestion
		if includeSBOM || outputSBOM != "" {
```
![image](https://user-images.githubusercontent.com/3977569/225154821-761eead7-2b87-4ed6-abef-bf799c129fe1.png)

Reply:
This *might* be worth having a test for as well...